drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -1785,6 +1785,9 @@ static int bcm2835_codec_create_component(struct bcm2835_codec_ctx *ctx)
 		goto destroy_component;
 
 	if (dev->role == ENCODE) {
+		u32 param = 1;
+		u32 param_size = sizeof(param);
```

Comment:
Why create a variable for this, rather than putting the sizeof at the point of use (c.f. line 1769)?

Reply:
Because my mind was thinking of ```vchiq_mmal_port_parameter_get``` where you have to pass a pointer to the size.
Ack.